tests/php/Stub/BasicMath/MethodRegisterHandler.php

```diff
@@ -34,7 +35,19 @@ public function start(StoreInterface $store)
      */
     public function register(HTTPRequest $request, StoreInterface $store)
     {
-        // Stub
+        $parameters = json_decode($request->getBody(), true);
+
+        if (empty($parameters['number'])) {
```

Comment:
This means that the number can't be zero - was that intentional?

Reply:
Nice catch!